lib/SPE.js

```diff
@@ -1321,9 +1318,7 @@ SPE.utils = {
         }
 
         newArray.push(
-            typeof srcArray[ sourceLength - 1 ].clone === 'function' ?
-            srcArray[ sourceLength - 1 ].clone() :
-            srcArray[ sourceLength - 1 ]
+          structuredClone(srcArray[ sourceLength - 1 ])
```

Comment:
You should revert that change.

Reply:
srcArray[ 0 ] was sometimes null, when setting attributes dynamically, and the old code would throw an exception.  I have reverted to the old code, but using optional chaining.

Reply:
It was several weeks ago, and I didn't record full details.  It happened when I was setting attributes programatically with an object.  I changed to setting the attributes with a string to fix a different problem, and I can't evoke the error with my current code.

Reply:
So that was probably an issue in your code calling one of the property with null I guess, so no need for the extra check here.